library/jruby

```diff
@@ -53,6 +53,14 @@ Tags: 9.4-jre21, 9.4.14-jre21, 9.4.14.0-jre21
 Architectures: amd64, arm64v8
 Directory: 9.4/jre21
 
+Tags: 9.4-jdk25, 9.4.14-jdk25, 9.4.14.0-jdk25
+Architectures: amd64, arm64v8
+Directory: 9.4/jdk25
+
+Tags: 9.4-jre25, 9.4.14-jre25, 9.4.14.0-jre25
+Architectures: amd64, arm64v8
+Directory: 9.4/jre25
+
 Tags: 9.3, 9.3.15, 9.3-jre, 9.3-jre8, 9.3.15-jre, 9.3.15-jre8, 9.3.15.0, 9.3.15.0-jre, 9.3.15.0-jre8
```

Comment:
JRuby 9.3 is EOL, right? https://github.com/jruby/jruby/wiki/Roadmap

If so, we should remove all the 9.3 tags from this file. :bow: :heart:

As a reminder, removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)